frontend/pages/g/_groupSlug/r/create/bookmarklet.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+    <div>
+      <v-form ref="domUrlForm">
+        <div>
+          <v-card-title class="headline"> {{ $t('recipe.create-bookmarklet') }} </v-card-title>
+          <v-card-text>
+            {{ $t('recipe.create-bookmarklet-description') }}
+            <v-checkbox v-model="importKeywordsAsTags" validate-on-blur hide-details :label="$t('recipe.import-original-keywords-as-tags')" />
+            <v-checkbox v-model="stayInEditMode" validate-on-blur hide-details :label="$t('recipe.stay-in-edit-mode')" />
+          </v-card-text>
+            <v-textarea
+            v-model="bookmarkletResult"
+            :label="$t('recipe.create-bookmarklet-result')"
+            :prepend-inner-icon="$globals.icons.tags"
+            filled
+            rows="2"
+            class="rounded-lg mt-2"
+            rounded
+            :hint="$t('recipe.create-bookmarklet-hint')"
+            persistent-hint
+            :readonly="true"
+            @click="copyBookmarklet"
+          ></v-textarea>
+        </div>
+      </v-form>
+    </div>
+  </template>
+
+  <script lang="ts">
+  import {
+    defineComponent,
+    reactive,
+    toRefs,
+    ref,
+    useRouter,
+    computed,
+    useContext,
+    useRoute
+  } from "@nuxtjs/composition-api";
+  import { detectServerBaseUrl } from "~/composables/use-utils";
+  import { VForm } from "~/types/vuetify";
+  import { useCopy } from "~/composables/use-copy";
+
+  export default defineComponent({
+    setup() {
+      const { $auth, req, i18n } = useContext();
+
+      const { copyText } = useCopy();
+
+      const route = useRoute();
+
+      const router = useRouter();
+
+      const importKeywordsAsTags = computed({
+        get() {
+          return route.value.query.use_keywords === "1";
+        },
+        set(v: boolean) {
+          router.replace({ query: { ...route.value.query, use_keywords: v ? "1" : "0" } });
+        },
+      });
+
+      const stayInEditMode = computed({
+        get() {
+          return route.value.query.edit === "1";
+        },
+        set(v: boolean) {
+          router.replace({ query: { ...route.value.query, edit: v ? "1" : "0" } });
+        },
+      });
```

Comment:
As those are not needed in the route, you can just implement them as normal variables
```suggestion
      const importKeywordsAsTags = ref<boolean>(false);
      const stayInEditMode = ref<boolean>(false);
```